locales/en.json

```diff
@@ -75,6 +76,8 @@
     "apps_catalog_obsolete_cache": "The app catalog cache is empty or obsolete.",
     "apps_catalog_update_success": "The application catalog has been updated!",
     "apps_catalog_updating": "Updating application catalog...",
+    "apps_failed_to_upgrade": "Those applications failed to upgrade:{apps}",
```

Comment:
```suggestion
    "apps_failed_to_upgrade": "Those applications failed to upgrade: {apps}",
```

Reply:
I'm don't think you want that because this is merged with the next line en en.json that will add a "\n" (but tbf it won't changed anything accepting or not your modification)

This is this output:

![image](https://user-images.githubusercontent.com/41827/220477378-6055b9f1-8d2a-42e3-9c80-304d67ba9ce8.png)

1 - is this docstring
2 - is `"apps_failed_to_upgrade_line": "\n * {app_id} (to see corresponding log do a 'yunohost log show {operation_logger_name}')",`